fractions/microprofile/microprofile-jwt/pom.xml

```diff
@@ -40,18 +38,6 @@
     </resources>
   </build>
 
-  <dependencyManagement>
```

Comment:
I'm going to remove this (and `arquillian.version` property) in a PR for SWARM-1627. Could you please leave it as it is so that we don't have to resolve conflicts?

Reply:
just this section or overall?

Reply:
Just this section.